Client/Frontend/Library/ReaderPanel.swift

```diff
@@ -348,17 +348,19 @@ class ReadingListPanel: UITableViewController, LibraryPanel {
     }
 
     override func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
-        guard let record = records?[indexPath.row] else { return nil }
+        guard let record = records?[safe: indexPath.row] else { return nil }
 
-        let deleteAction = UIContextualAction(style: .destructive, title: .ReaderPanelRemove) { [weak self] (_, _, completion) in
+        let deleteAction = UIContextualAction(style: .destructive,
+                                              title: .ReaderPanelRemove) { [weak self] (_, _, completion) in
             guard let strongSelf = self else { completion(false); return }
-
+            
```

Comment:
Swiftlint: whitespace?

Reply:
Argh. I messed up my config in Xcode. Let me fix that